maestro-studio/web/src/helpers/models.ts

```diff
@@ -38,7 +38,6 @@ export type ReplCommand = {
 }
 
 export type Repl = {
-  version: number
```

Comment:
This version was only used for long polling. No longer necessary now that we migrated to SSE